docs/Developers/Development_Setup.rst

```diff
@@ -83,21 +83,15 @@ master branch from and all of its dependencies with pip using
 
 ::
 
+    (coala-venv)$ # For the coala repository:
     (coala-venv)$ git clone https://github.com/coala/coala
-    (coala-venv)$ cd coala
-    (coala-venv)$ pip3 install -e .
-    (coala-venv)$ cd -
+    (coala-venv)$ pip3 install -e coala
+    (coala-venv)$ # For the coala-bears repository:
     (coala-venv)$ git clone https://github.com/coala/coala-bears
-    (coala-venv)$ cd coala-bears
-    (coala-venv)$ pip3 install -e .
+    (coala-venv)$ pip3 install -e coala-bears
 
-Then you can install a repository-backed version of the repository you would
-like to modify using
-
-::
-
-    (coala-venv)$ pip3 install -e <path/to/clone>
 
+This will install the repository in editable mode.
 You will then be able to edit the repository and have the changes take effect
```

Comment:
Maybe connect these two sentences nicely: "This will install the repository in editable mode, meaning you will be able to edit the repository and have the changes take effect [...]"